tests/e2e/parameters/test_appconfig.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+import pytest
+
+from tests.e2e.utils import data_fetcher
+
+
+@pytest.fixture
+def parameter_appconfig_freeform_handler_fn_arn(infrastructure: dict) -> str:
+    return infrastructure.get("ParameterAppconfigFreeformHandlerArn", "")
+
+
+@pytest.fixture
+def parameter_appconfig_freeform_handler_fn(infrastructure: dict) -> str:
+    return infrastructure.get("ParameterAppconfigFreeformHandler", "")
+
+
+@pytest.fixture
+def parameter_appconfig_freeform_value(infrastructure: dict) -> str:
+    return infrastructure.get("AppConfigConfigurationValue", "")
+
+
```

Comment:
Should we have a test for FeatureFlags ConfigurationProfile type too?

I don't recall if we have full control on the content of that type in AppConfig (as in, add our expected schema). If we do, it'd be contribute to a more comprehensive coverage to detect this feature will work in both scenarios.

PS: The fetching will be exactly the same, so your Lambda function can be reused (remove `freeform` from the handlers name)

Reply:
@heitorlessa currently we don't support [AWS.AppConfig.FeatureFlags](https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-creating-configuration-and-profile.html#appconfig-type-reference-feature-flags) profile and I added a info in [documentation](https://github.com/awslabs/aws-lambda-powertools-python/pull/1553/files#diff-0e8fe1a0c18945a2f28b5a87df6c635fbbbfc7d87e44305a0978e41db488024bR8)